src/main/scala/zio/prelude/Associative.scala

```diff
@@ -1272,4 +1272,24 @@ trait AssociativeSyntax {
     def combine(r: => A)(implicit associative: Associative[A]): A =
       associative.combine(l, r)
   }
+
+  /**
+   * Provides syntax for combining values in a `ParIterable` which is thus done in a parallel manner.
+   */
+  implicit class AssociativeParIterableOps[A](private val p: collection.parallel.immutable.ParIterable[A]) {
+
+    /**
+     * Associatively combines the values in a parallel manner,
+     * while blocking the thread.
+     */
+    def reduceAssociativeBlocking(implicit associative: Associative[A]): Option[A] =
```

Comment:
Why the name 'reduceAssociativeBlocking' for this one?

Reply:
Nevermind, I see why. I suggest `reduceAssociative` for this one, and `reduceAssociativeM` for the other one, keeping with the ZIO convnetion of `M` = effectful variant.

Reply:
:heavy_check_mark: